app/assets/stylesheets/home.scss

```diff
@@ -37,7 +37,7 @@ h3 {
 
 .sign-in a {
   text-align: center;
-  background-color: #34495e;
+  background-color: #00796B;
```

Comment:
Color literals like `#00796B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#00796B` should be written as `#00796b`